frontend/simple/assets/sass/theme/kindergarten-professional.scss

```diff
@@ -1,4 +1,4 @@
-$size-6: 0.875rem; // 14px
+$size-6: 0.875rem; // 14px // REVIEW - isn't this value too small on mobile?
```

Comment:
I don't know? Where does this appear? Screenshot? (I personally think that smaller is better! :D)

Reply:
Personally I like it smaller too... but... I'm concern if our users will like it too. Run GI in your iPhone and compare the chat text size with for example Slack or Discord, the text is not the standard size and maybe it should be.

Reply:
Yeah I haven't tested it yet on the iPhone. I'm guessing Slack and Discord do the right thing, so feel free to do what they do, unless you disagree and think you have a better answer. :)

Reply:
Code updated with specific size just for phone, check `_groupincome.scss`